### PR TITLE
Consume TranscendAPI

### PR DESCRIPTION
**Pull request by JonnavithulaGirish**
## Change log
- partially closes [T-31931](https://transcend.height.app/T-31931)
- partial closes [T-31932](https://transcend.height.app/T-31932)

### Internal change log
- Check if backend/hidden webView is working
- Consume getRegimes and getConsent

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -57,12 +58,12 @@
...
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:visibility="visible"
-        app:transcendConsentUrl= "@string/transcendConsentUrl"/>
+<!--    <io.transcend.webview.TranscendWebView-->
+<!--        android:id="@+id/webView"-->
+<!--        android:layout_width="match_parent"-->
+<!--        android:layout_height="match_parent"-->
+<!--        android:visibility="visible"-->
+<!--        app:transcendConsentUrl= "@string/transcendConsentUrl"/>-->
```
Comment (MadDataScience) [inline app/src/main/res/layout/activity_main.xml:66]: Should this be removed?

File: app/build.gradle [hunk 2]
```diff
@@ -8,7 +8,7 @@ android {
 
     defaultConfig {
         applicationId "io.transcend.samplesdk"
-        minSdk 24
+        minSdk 21
```
Comment (MadDataScience) [inline app/build.gradle:11]: How did minSdk go down?

**Review by MadDataScience — COMMENTED** [review]
Is it possible/feasible to add unit tests for this?

File: app/build.gradle (hunk 2 above)
Comment (JonnavithulaGirish) [reply app/build.gradle:11]: Athlectic wanted minSDK>=21 so, we have updated our library to be compatable.

**Comment by JonnavithulaGirish** [comment]
> Is it possible/feasible to add unit tests for this?

ISince this is a sampleApp, I guess we could avoid Unittests here and can plan on adding CPU and Memory profile tests.

**Comment by anotherminh** [comment]
@MadDataScience yeah it doesn't make sense to add unit tests for this IMO -- I would prioritize adding tests for the actual library

**Review by anotherminh — APPROVED** [review]

**Pull request merged**